lib/simpleRequester.js

```diff
@@ -14,7 +14,18 @@ exports.do = options => {
         reject(error)
         return
       }
-      if (res.statusCode !== 200) {
+      if (res.statusCode === 401) {
```

Comment:
If you want to get additional information on the JSON parse error we get for the issues endpoint I'm afraid that it won't have any effect because we are returning an empty string. I think that instead of polling the issues endpoint we should poll the analysis endpoint and parse the status.

Reply:
Thanks for trhe information. I think will have to come back to redo this as well - later.

Reply:
The 401 handling though does help a little (and that's what I encountered).

Reply:
For 401 invalid authentication this is better. But yes we may need to go back and redo this. But later for now. The 401 problem is what I encountered which is why it was addressed this way.